src/Sylius/Bundle/TaxonomyBundle/Resources/config/doctrine/model/Taxon.orm.xml

```diff
@@ -56,6 +56,8 @@
         </field>
 
         <gedmo:tree type="nested" />
+
+        <field name="enabled" column="enabled" type="boolean" />
```

Comment:
redundant `column` attribute

Reply:
But we have the practice to leave them in Core